jest.config.js

```diff
@@ -0,0 +1,20 @@
+const config = {
+  "preset": "ts-jest/presets/js-with-ts",
+  "setupFiles": [
+    "<rootDir>/src/__helpers__/setupEnvVars.js"
+  ],
+  "testEnvironment": "jsdom",
+  "coverageThreshold": {
+    "global": {
+      "lines": 90
+    }
+  },
+  "transformIgnorePatterns": ["node_modules\/(?!axios)"],
+  "moduleNameMapper": {
+    // Force CommonJS build for http adapter to be available.
+    // via https://github.com/axios/axios/issues/5101#issuecomment-1276572468
+    '^axios$': require.resolve('axios'),
```

Comment:
See config requirement here: https://github.com/nock/nock?tab=readme-ov-file#axios

This setting doesn't work in package.json, so I had to move the config to a new file